rpm/patterns-sailfish-api-3.spec

```diff
@@ -0,0 +1,47 @@
+Summary: Sailfish API Level 3
```

Comment:
This all should be added just a subpackage of the main .spec file so that we would have only one webhook to build it all, needing two webhooks is a bit messy. Also both hooked packages would anyway have e.g. same changelog.